### PR TITLE
Post  deps

### PR DESCRIPTION
**Pull request by tadatuta**
cc @tormozz48 @tavriaforever

### CODE REVIEW

**Comment by tadatuta** [comment]
it's not tested and for sure still buggy if used with short form in `bower.json`

**Comment by tadatuta** [comment]
any reason not to merge?

File: src/blocks/common.blocks/post/_view/post_view_full.bemtree [hunk 1]
```diff
@@ -9,8 +9,6 @@ block post, mod view full {
             return [];
         }
 
-        //console.log('url %s deps %s', source.url, source.deps);
```
Comment (tavriaforever) [inline src/blocks/common.blocks/post/_view/post_view_full.bemtree:12]: This is not an error, it is left for the application

File: src/blocks/common.blocks/post/__deps/post__deps.bemhtml [hunk 2]
```diff
@@ -1,34 +1,33 @@
 block('post').elem('deps')(
-    match(function() { return !(this.ctx && this.ctx.content); }).def()(function() {
-        return false;
-    }),
+    match(function() { return !(this.ctx && this.ctx.content); }).def()(),
```
Comment (tavriaforever) [inline src/blocks/common.blocks/post/__deps/post__deps.bemhtml]: Missed `false`-> .def()(false)

**Comment by tavriaforever** [comment]
PR was rebased and now it ready to merge! @tadatuta thanks!

**Pull request merged**